mappings/maestro_analysis_centric.json

```diff
@@ -82,7 +82,7 @@
           "platform": {
             "type": "keyword"
           },
-          "library_strategy": {
```

Comment:
do the same for the platform file centric index ?

Reply:
this affects rdpc, we should include Alexis/Alex too or post this in rdpc dev

Reply:
you already did post this, thanks

Reply:
resolving this comment as 
- it is changed below 
- i will make sure on slack both alex and alexis see the change, in addition to the current post.